osquery/tables/system/windows/system_util.cpp

```diff
@@ -89,36 +89,95 @@ std::string WmiResultItem::GetString(const std::string& name) const {
   return BSTR_to_string(value.bstrVal);
 }
 
+// Returns a pair with the first value indicating success of the query
+// and the second value being the data obtained from the query.
```

Comment:
For multi-line function docs use the longer /* block comment will a brief, adding the params and return is helpful too :D